docs/mina-protocol/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Mina Protocol
+sidebar_label: Overview
+hide_title: true
+description: This section describes how the Mina Protocol works
+keywords:
+  - mina protocol
+  - block producers
+  - block
+  - snark workers
+  - scan state
+  - hard forks
+  - time locked accounts
+  - sending a payment
+  - lifecycle of a payment
+---
+
+# Introduction
+
+The Mina Protocol is a layer one protocol designed to deliver on the original promise of blockchain, true decentralization, scale and security. 
+
+Mina offers an elegant solution: replacing the blockchain with an easily verifiable, consistent-sized cryptographic proof. Mina dramatically reduces the amount of data each user needs to download. Instead of verifying the entire chain from the beginning of time, participants fully verify the network and transactions using recursive zero knowledge proofs (or zk-SNARKs). Nodes can then store the small proof, as opposed to the entire chain. Because it’s a consistent size, Mina stays accessible even as it scales to many users and accumulates years of transaction data.
+
+## The Mina Protocol 
+
+Currently there are three public Mina Protocol networks:
+
+1. Mainnet - the production network
+2. Devnet - the test network based on the same software versions as the mainnet
+3. Berkley Test Network - a development network, where new features are trialed
+
+In this section we describe how the Mina Protocol works.
```

Comment:
```suggestion
This section describes how the Mina Protocol works.
```

Reply:
@halsaphi  
[Use second person](https://developers.google.com/style/person): "you" rather than "we."